templates/article.jade

```diff
@@ -0,0 +1,25 @@
+
+extends layout
+
+block append vars
+  - bodyclass = 'article-detail'
+
+block prepend title
+  | #{ page.title + " - "}
+
+block header
+  include author
+  h1= page.title
+  p.author
+    | #{ "Written by " }
```

Comment:
Or keep it consistent with `| #{"Written by"}` but this seems like a lot of excess